site/operational-guide/tasks.md

```diff
@@ -54,41 +65,41 @@ The `weave rmpeer` command is provided to perform this task, and must
 be executed on _one_ of the remaining peers. That peer will take
 ownership of the freed address space.
 
-##<a name="cluster-upgrade"></a>Upgrade a Cluster
+##<a name="cluster-upgrade"></a>Upgrading a Cluster
 
 Protocol versioning and feature negotiation are employed in Weave Net
-to enable incremental rolling upgrades - each major release maintains
+to enable incremental rolling upgrades. Each major maintains
 the ability to speak to the preceding major release at a minimum, and
-connected peers only utilise features which both support. The general
-upgrade procedure is as follows:
+connected peers only utilize features which both support. 
+
+The general upgrade procedure is as follows:
 
-On each peer in turn:
+On each peer:
 
-* Stop the old weave with `weave stop` (or `systemctl stop weave` if
+* Stop the old Weave with `weave stop` (or `systemctl stop weave` if
   you're using a systemd unit file)
-* Download the new weave script and replace the existing one
-* Start the new weave with `weave launch <existing peer list>` (or
+* Download the new Weave script and replace the existing one
+* Start the new Weave with `weave launch <existing peer list>` (or
   `systemctl start weave` if you're using a systemd unit file)
 
-This will result in some downtime as the first launch with the new
-script has to pull the new container images; if you wish to minimise
-downtime you can download the new script to a temporary location
-first:
+Since the first launch with the new script pulls the new container images, this 
+results in some downtime. To minimize downtime, download the new script 
+to a temporary location first:
```

Comment:
The second half of that last sentence is redundant, since it's the same as the first item in the list that follows.